### DISCUSSION TITLE
Is it possible to support ARM64 binary image in "Copying bpftrace binary from docker" section?

### DISCUSSION THREAD

**Discussion by vipwangerxiao** [bpftrace/bpftrace#2314] [Q&A] [answered]
Now there is a x86 bpftrace binary, and can be easily pulled as bpftarce/INSTALL.md says:

"docker pull quay.io/iovisor/bpftrace:master-vanilla_llvm_clang_glibc2.23"

QUESTION:

where can arm64 binary be found?

Thanks a lot!

**Comment by danobi** [comment, marked as answer]
We would probably need to add support for an arm build. Probably not very high on anyone's priorities at the moment, unfortunately.

Your best bet is a build from source.

**Comment by gsingh93** [comment]
I haven't tried it yet, but this claims to be able to cross-compile an ARM binary (for Android): https://github.com/facebookexperimental/ExtendedAndroidTools
